src/main/webapp/resources/js/renderCalendar.js

```diff
@@ -695,7 +762,25 @@ function clearEventDialogSingleMulti(){
     $('#color-select').val("#1ba1e2");
     $('#event-title').val("");
     $('#description').val("");
+}
 
+function buildTableMonthly() { //generating table for DAYS_IN_MONTH day
+    var tableHTML = '';
+    var dayNum = 0;
+    var monthlyTable = document.getElementById("monthly-days");
```

Comment:
Why you mix js and jquery
use $("#monthly-days") instead document.getElementById("monthly-days");